models/src/anemoi/models/layers/utils.py

```diff
@@ -31,20 +32,30 @@ def forward(self, *args, **kwargs):
         return checkpoint(self.module, *args, **kwargs, use_reentrant=False)
 
 
-def load_layer_kernels(kernel_config: DotDict) -> DotDict:
+def load_layer_kernels(kernel_config: Optional[DotDict] = None, instance: bool = True) -> DotDict["str" : nn.Module]:
     """Load layer kernels from the config.
 
-    Args:
-        kernel_config : DotDict
-            Kernel configuration, e.g. {"Linear": {"_target_": "torch.nn.Linear", "_partial_": True}}
+    This function tries to load the layer kernels from the config. If the layer kernel is not supplied, it will fall back to the torch.nn implementation.
 
-    Returns:
-        DotDict: hydra partial instantiation of the layer kernels
+    Parameters
+    ----------
+    kernel_config : DotDict
+        Kernel configuration, e.g. {"Linear": {"_target_": "torch.nn.Linear", "_partial_": True}}
```

Comment:
Does the `kernel_config` still need the '_partial_' or should we update this line?